x-pack/test_serverless/shared/config.base.ts

```diff
@@ -137,6 +137,8 @@ export default async () => {
         })}`,
         '--xpack.encryptedSavedObjects.encryptionKey="wuGNaIhoMpk5sO4UBxgr3NyW1sFcLgIf"',
         `--server.publicBaseUrl=${servers.kibana.protocol}://${servers.kibana.hostname}:${servers.kibana.port}`,
+        // disable fleet task that writes to metrics.fleet_server.* data streams, impacting functional tests
+        `--xpack.task_manager.unsafe.exclude_task_types=${JSON.stringify(['Fleet-Metrics-Task'])}`,
```

Comment:
Can you please explain the impact on functional tests? What happens without this setting?

Asking because we don't want to add this kind of configuration to serverless tests outside the feature flag testing. The reason for that is, that when we create a real serverless project in MKI, this setting would not be applied but the tests would still need to pass there (the config files are only controlling the local setup).
So if serverless tests are failing without this setting, then they would most probably still fail when run as part of our release gates on an MKI project and thus we'd need a different solution here.

Reply:
It seems that the same test is [still failing](https://buildkiteartifacts.com/e0f3970e-3a75-4621-919f-e6c773e2bb12/0fda5127-f57f-42fb-8e5a-146b3d535916/018b41ab-4cc1-4628-9a74-a342fa2b9931/018b41ad-28c8-45d3-9e87-262298d7b29e/target/test_failures/018b41ad-28c8-45d3-9e87-262298d7b29e_94c67289d3f5a902be9032da61d15325.html?response-content-type=text%2Fhtml&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=ASIAQPCP3C7LWRHDZXMQ%2F20231018%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20231018T091342Z&X-Amz-Expires=600&X-Amz-Security-Token=IQoJb3JpZ2luX2VjEIf%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJHMEUCIQDtLasVFLfjCYfx%2B2v0Bj%2By03Ifhmz4l0vtizcic6bJsgIgYmR%2Bw9mw9pv2ppzpYFMFReArU%2FOm3Zi8OYkwz5Pj95Mq%2BgMIoP%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FARAAGgwwMzIzNzk3MDUzMDMiDHy4gyWmMpSdAQLMpSrOA7OdLz%2F%2BMJYKLsnrCjK6oPj1BNrtHaJNCHg%2BGjV0P1SD%2FafavnXCEhOfQLZqXIeCYFfDJcaYU%2FZVigybaaA4rYZJEyMmrNLsuchhiVEfuNA40ft%2F5bYBZf3J1YxU7uwbI7MPB8Nh%2FBoRh9gvmPSVayyt87HUMhvbKPFq2sBE7Bm7DmLFDu4V88vUKnkGjJBgFv3PJ9cKT%2FH3i8UQ9ZNc1wJz2RZLEHLfJASn%2BCzJ3sve6Pij0oyjil85wmUAj5AWP2ZhAZTNGk5LPxE%2FZYOXnd3N%2BAYkYO2A7WV0yS1xusDvayeTKvWnF0pw4i2W8NJ0%2FV%2BJjpiSPlVPjWcJZaIfafpiOUCLutYr8A35LKS9klNvILsBanFHu%2F4vwjHU24X56CbSL3TpVRUBWY0n%2BZQbFY0ot9%2BF5HvBBWA6x3EWheGSPO0l121iBMsbPxHVzkBgsPMXg%2F%2BidDAUw4HgpAL9QjZMVa7CPc0vt0UL6UYfhAM70J%2Fszl%2FfqCdeipEhYpdD6J%2FvJKxtzMUU6OP%2Fdvr8oVK91%2BPDWiaHzwWxvXZI%2F4gelb8sxADcjsB%2F4SirtN5UlMHJ8rkpmnaU2hDcrcyYmd1%2Fp4SBPA%2BDCXlLMawMzDDq%2Br2pBjqlAQxy8qHtHGw7EW1at6Q%2BK2R3PgjewzdwsJU%2FCWOpqk6IZMDZ0IGxA8CuBWzAen%2BeiSHR0uy2sF4QXGEoGHoH57cFdL9aHSfSKy%2FF1Z1KNHpprIe2C042cq6GTKSt2MMdmDp4v%2B9Y6lQYDsqmbvjiM69Wb04kmJrzkPDvgXwqvJMIS%2BsCk6y%2F60A8vaOqcRCwP6KP5c5AUyd85aC1wbv0h4qqZwx%2Few%3D%3D&X-Amz-SignedHeaders=host&X-Amz-Signature=c9afd3a21ab95361bc7b8a616c4bda1759be527d9f0dc940fb004cff939cd285) even when disabling this task, so it's not the root cause of the issue, I can revert it.

It seems that my changes are not related to the test failing, should I skip it to pass the build? It already has an open issue: https://github.com/elastic/kibana/issues/166592

Reply:
reverted the config change and skipped the failing test

Reply:
It's a different failure this time than reported in #166592.
Also, I don't see this test failing on the main branch recently, so there's a good chance the failure is really related this the changes in this PR.

Reply:
I found this label though saying it fails in MKI:
https://github.com/elastic/kibana/blob/7a6826be3f3352944f887591bb439d9dd5ed9ed2/x-pack/test_serverless/functional/test_suites/observability/cases/attachment_framework.ts#L23-L25

Reply:
@pheyos The failures on this PR seem unrelated to the issue but it is the first time I am seeing them. FWIW, I am working on this PR https://github.com/elastic/kibana/pull/168924 to fix some issues around the Cases Serverless tests.

Reply:
Also found the same (?) test skipped under `security`:
https://github.com/elastic/kibana/blob/6d88fb5f54d4c1fbd8ff1b41d3bfcc4b3f244bed/x-pack/test_serverless/functional/test_suites/security/ftr/cases/attachment_framework.ts#L21-L23

Reply:
Yes, this is skipped for different reasons. I am not in favor of skipping tests outside of the standard process but given that I am working on them I am okay with this being skipped.